Sources/MongoSwift/BSON/BsonValue.swift

```diff
@@ -185,16 +167,34 @@ public struct Binary: BsonValue, Equatable, Codable {
     public let data: Data
 
     /// The binary subtype for this data.
-    public let subtype: BsonSubtype
+    public let subtype: Subtype
+
+    /// Subtypes for BSON Binary values.
+    public enum Subtype: Int, Codable {
+        /// Generic binary subtype
+        case binary = 0,
+        /// A function
+        function,
+        /// Binary (old)
+        binaryDeprecated,
+        /// UUID (old)
+        uuidDeprecated,
+        /// UUID
+        uuid,
+        /// MD5
+        md5,
+        /// User defined
+        user
```

Comment:
"user" may be a bit ambiguous. "userDefined" may be clearer.

Separate note: although http://bsonspec.org/spec.html does define a "user defined" type as 0x80, technically everything in the range of 0x80 to 0xFF is reserved for user-defined usage.